resources/wiki-overrides.properties

```diff
@@ -47,9 +41,6 @@ prereq-buildstep=http://wiki.jenkins-ci.org/display/JENKINS/Prerequisite+build+s
 project-health-report=https://wiki.jenkins-ci.org/display/JENKINS/Project+Health+Report+Plugin
 puppet=https://wiki.jenkins-ci.org/display/JENKINS/Puppet+Plugin
 ruby-runtime=https://wiki.jenkins-ci.org/display/JENKINS/Ruby+Runtime+Plugin
-serenity=https://wiki.jenkins-ci.org/display/JENKINS/Serenity+Plugin
```

Comment:
https://github.com/jenkinsci/serenity-plugin/commit/ef4c5b00f4638fece63b3f7e31640d6298b76818 references the repo now, but docs were migrated.